✨(BuildConfig) force to pull last docker images

lunika opened the pull request:
## Purpose

Marsha and Richie use the `master` docker image on staging environment. We want to force the build to force the latest available docker image without having to deploy new  `BuildConfig` object


## Proposal

- [x] set a `dockerStrategy` in the BC template to force pull and no cache.

lunika committed:
✨(BuildConfig) force to pull last docker images

On marsha and Richie applications we want to force the Build to pull the
last available image without any cache. With this we can deploy the last
master version without having to change the configuration.

jmaupetit commented on CHANGELOG.md:
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 
 - Python dependencies have been upgraded thanks to pyup bot.
+- Marsha and richie BuildConfig force to pull the last available docker image.
```
```suggestion
- Marsha and richie BuildConfig force to pull the latest available docker image.
```

jmaupetit replied:
I've fixed this.

jmaupetit committed:
👷(circle) remove docker cache

Since we are constantly testing against master-tagged images for Richie
and Marsha, it is safer to prevent job failure due to a cache that
should have been invalidated.
